@kiva/kv-components/src/vue/KvSideSheet.vue

```diff
@@ -176,6 +181,25 @@ export default {
 			type: String,
 			default: '',
 		},
+		/**
+		 * The CSS width of the SideSheet, either as a string (e.g., '33.33%') or an object with
+		 * breakpoint-specific widths (e.g., { default: '100%', lg: '50%' }). Supports responsive
+		 * design with breakpoints: default, sm, md, lg, xl, 2xl
+		 * */
+		widthDimensions: {
```

Comment:
Please add a story that passes in some breakpoint values so we can see it live in storybook.